include/debug.h

```diff
@@ -0,0 +1,17 @@
+#include <string>
+
+#include "buffer.h"
+
+namespace celerity {
+namespace debug {
+	template <typename DataT, int Dims>
+	void set_buffer_name(const celerity::buffer<DataT, Dims>& buff, const std::string& debug_name) {
+		detail::runtime::get_instance().set_buffer_debug_name(detail::get_buffer_id(buff), debug_name);
```

Comment:
Why is there a dedicated function for setting the debug name, but not for getting it? In any case, I'm in favor of just using `get_buffer_manager`.